README.adoc

```diff
@@ -69,7 +69,7 @@ For more installation options please refer the link:docs/installation.adoc[insta
 In order to correctly use `odo` you must download it and add it
 to your `PATH` environment variable:
 
-. Download the `odo-windows-amd64.exe.gz` file from the
+. Download the `odo-windows-amd64.exe.tar.gz` file from the
```

Comment:
@amitkrout this change

Reply:
Ok, then it will comes to effect only when the very next release happen. Correct me if i am wrong

Reply:
docs merged in would directly show up on github, and I think isn't that the place where people go to for docs? I dont think  `we release docs` on a separate website. so if I merge this now people will see this as the current docs.